pkg/packet/bgp/internal/generate.go

```diff
@@ -51,7 +51,7 @@ func generateSRBehaviour() {
 	buf.Write(constants)
 
 	if err := os.WriteFile(srBehaviourTargetFile, buf.Bytes(), 0o664); err != nil {
-		doPanic("failed to write file %s: %w", )
+		doPanic("failed to write file %s: %w", srBehaviourTargetFile, err)
```

Comment:
What's this change? The generated code has a bug?

Reply:
Oh I just wanted to re-trigger CI, and this looked like a typo in existing code so I fixed it:
https://github.com/osrg/gobgp/blob/3ad3adb76aed1acbd949992fa52d8bef08f41124/pkg/packet/bgp/internal/generate.go#L57-L59

I can revert if you like.

Reply:
Looks like this is generated code so we should not touch.

Reply:
ok reverted.